### PR TITLE
update of scripts.

### PR DESCRIPTION
**Pull request by istoimenova**

### CODE REVIEW

**Comment by istoimenova** [comment]
@dboltovskyi, please review.

File: test_scripts/Polices/Policy_Table_Update/026_ATF_P_TC_PTU_Merge_Into_LPT.lua [hunk 1]
```diff
@@ -59,7 +60,31 @@ local function is_table_equal(t1, t2)
   return true
 end
 
+local function update_preloaded_pt_removeRC()
+  commonPreconditions:BackupFile("sdl_preloaded_pt.json")
```
Comment (dboltovskyi) [inline test_scripts/Polices/Policy_Table_Update/026_ATF_P_TC_PTU_Merge_Into_LPT.lua]: @istoimenova, It's better to move out back up of sdl_preloaded_pt.json file from the function

Comment (istoimenova) [inline test_scripts/Polices/Policy_Table_Update/026_ATF_P_TC_PTU_Merge_Into_LPT.lua]: @dboltovskyi, done in e86df3a59d57d906ea180fb5a2efa2385d49ee0e

**Comment by dboltovskyi** [comment]
@istoimenova, Thank you for update

**Pull request merged**